sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClient.cs

```diff
@@ -1592,6 +1593,62 @@ public virtual VerifyResult VerifyData(SignatureAlgorithm algorithm, Stream data
             }
         }
 
+        /// <summary>
+        /// Creates an <see cref="RSA"/> implementation backed by this <see cref="CryptographyClient"/>.
+        /// </summary>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> to cancel this operation.</param>
+        /// <returns>An <see cref="RSA"/> implementation backed by this <see cref="CryptographyClient"/>.</returns>
+        /// <remarks>
+        /// The <see cref="CryptographyClient"/> will attempt to download the public key synchronously.
+        /// </remarks>
+        /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
+        public virtual RSA CreateRSA(CancellationToken cancellationToken = default)
```

Comment:
@KrzysztofCwalina @tg-msft @schaabs there are separate sync and async calls for the initial fetch attempt of the public key; otherwise, waiting to fetch inside the crypto primitive will always be sync over network.

Also, is returning the abstract base `RSA` here fine? One normally should not interact with the derivatives.